MonoGame.Framework.Content.Pipeline/Builder/PipelineManager.cs

```diff
@@ -458,6 +506,26 @@ private PipelineBuildEvent LoadBuildEvent(string destFile, out string eventFilep
             return PipelineBuildEvent.Load(eventFilepath);
         }
 
+        public void RegisterContent(string sourceFilepath, string outputFilepath = null, string importerName = null, string processorName = null, OpaqueDataDictionary processorParameters = null)
+        {
+            sourceFilepath = PathHelper.Normalize(sourceFilepath);
+            ResolveOutputFilepath(sourceFilepath, ref outputFilepath);
+
+            ResolveImporterAndProcessor(sourceFilepath, ref importerName, ref processorName);
+
+            var contentEvent = new PipelineBuildEvent
+            {
+                SourceFile = sourceFilepath,
+                DestFile = outputFilepath,
+                Importer = importerName,
+                Processor = processorName,
+                Parameters = ValidateProcessorParameters(processorName, processorParameters),
+            };
+
+            // Register pipeline build event. (Required to correctly resolve external dependencies.)
+            TrackPipelineBuildEvent(contentEvent);
```

Comment:
I suspect some processors/importers internally generate new content items to process.  How do those work out as far as consistency of processing order?